pkg/api/apiloader.go

```diff
@@ -295,11 +294,6 @@ func setContainerServiceDefaultsv20170131(c *v20170131.ContainerService) {
 }
 
 // Sets default HostedMaster property values for any appropriate zero values
-func setManagedClusterDefaultsv20170831(hm *v20170831.ManagedCluster) {
-	hm.Properties.KubernetesVersion = ""
-}
-
-// Sets default HostedMaster property values for any appropriate zero values
 func setManagedClusterDefaultsvlabs(hm *apvlabs.ManagedCluster) {
```

Comment:
Can you remove this as well?

Reply:
Dmitry will have the change in his PR. Can you leave a comment to him. Yes, we should also remove this.

Reply:
closed in favor of #1578